back/spec/acceptance/initiative_comments_spec.rb

```diff
@@ -298,6 +302,10 @@
           assert_status 401
           expect(json_response_body.dig(:errors, :base, 0, :error)).to eq 'Unauthorized!'
         end
+
```

Comment:
Do we need to remove author_id as a parameter in post/patch requests as we have done in the idea comments spec?

Reply:
Weirdly, the author parameter was never supported for commenting, but it was specified as a parameter in the specs. So there was nothing to be removed.